src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import type { FC } from 'react';
+
+import type { Todo } from '../../types/Todo';
+
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map((todo) => (
+      <TodoInfo key={todo.id} todo={todo} />
+    )).sort((a, b) => a.props.todo.id - b.props.todo.id)}
```

Comment:
i'd start with `.sort` and then chain `.map` - more neat this way
```suggestion
    {
      todos
        .sort((a, b) => a.id - b.id)
        .map((todo) => (
          <TodoInfo key={todo.id} todo={todo} />
        ))
    }
```